rules/xcodeproj.bzl

```diff
@@ -87,10 +87,11 @@ def _make_swift_vfs_args(vfs_path):
 def _make_swift_copts(target, deps, ctx):
     hmap_files = []
     vfs_files = []
+    collected_vfs = False
     for dep in deps:
-        if VFSOverlayInfo in dep:
+        if VFSOverlayInfo in dep and not collected_vfs:
```

Comment:
So we're assuming that only one `dep` will hold the vfs provider right? Is that always the case or is it possible that more than one `dep` could be holding this provider?

Reply:
Yeah exactly, the existing code did that so was going off that 👍